web/src/common/SequentialLegendBucketed.svelte

```diff
@@ -0,0 +1,22 @@
+<script>
```

Comment:
This is a fork of SequentialLegend from svelte-utils, but the labels fall **on** the buckets rather than **between** the buckets (consequently, there's one less label).

Reply:
Nice! I might upstream this and use in NPW too.

Missing TS types, but I'll add them after merging the PR stacks to avoid any rebase headache.